packages/manager/apps/pci-billing/vite.config.ts

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from 'vite';
+import { getBaseConfig } from '@ovh-ux/manager-vite-config';
+import tailwindcss from 'tailwindcss';
+
+const baseConfig = getBaseConfig({});
+
+export default defineConfig({
+  ...baseConfig,
+  root: '',
+  css: {
+    postcss: {
+      plugins: [tailwindcss],
+    },
+  },
+  resolve: {
+    ...baseConfig.resolve,
+    dedupe: [
+      'i18next',
+      'react',
+      'react-dom',
+      'react-i18next',
+      'react-router-dom',
+      'zustand',
+    ],
```

Comment:
```suggestion
    dedupe: [
      'i18next',
      'react',
      'react-dom',
      'react-i18next',
      'react-router-dom',
      'zustand',
    ],
```
this can be removed as it is now part of base config